Formula/pldiff.rb

```diff
@@ -0,0 +1,16 @@
+class Pldiff < Formula
+  desc "Compares plist files line by line"
+  homepage "https://github.com/scottrigby/pldiff"
+  url "https://github.com/scottrigby/pldiff/archive/1.0.4.tar.gz"
+  sha256 "0093027aee4ceb049902a9fcea28923477c87c2f88813e2370fb4c12195b9ccd"
+
+  depends_on "colordiff" => :recommended
+
+  def install
+    system "/bin/sh", "install.sh", prefix
```

Comment:
Yeah, just `sh` works here!

The choice between `prefix` and `#{prefix}` really just depends on whether you're using it for interpolation or not. In your case, bare `prefix` is fine since it's a separate argument. If it were being inserted into an argument (e.g. `Pathname#/`: `prefix/"foo"`), we prefer the interpolation style (`"#{prefix}/foo"`).